runtime/src/chpl-privatization.c

```diff
@@ -71,8 +72,22 @@ void chpl_newPrivatizedClass(void* v, int64_t pid) {
 }
 
 
-extern void* chpl_getPrivatizedClass(int64_t i) {
+void* chpl_getPrivatizedClass(int64_t i) {
   return chpl_privateObjects[i];
 }
 
 
+void chpl_clearPrivatizedClass(int64_t i) {
+  chpl_privateObjects[i] = NULL;
+}
+
+// Used for to check for leaks of privatized classes
+int64_t chpl_numPrivatizedClasses(void) {
```

Comment:
right, clearPrivatizedClass is called on normal code paths, but I doubt acquiring the lock there will have significant impact.